py/visualize.py

```diff
@@ -288,7 +295,8 @@ def wrap_labels(labels, n):
                     x=xd[0] / 2,
                     y=1.15,
                     text=top_labels[0],
-                    font=dict(family="Arial", size=14, color="rgb(67, 67, 67)"),
+                    font=dict(family="Arial", size=14,
```

Comment:
Could we use Roboto throughout?

Reply:
Here's Arial:
![image](https://user-images.githubusercontent.com/16396180/133514220-67fc38d0-9a26-4885-99d6-e858b3ba6c12.png)

and here's Mr. Roboto:
![image](https://user-images.githubusercontent.com/16396180/133514489-af9974e4-e0c2-4cba-9330-058b596eeff6.png)

Arial looks better on the charts IMO